src/test/java/uk/gov/companieshouse/web/accounts/service/notehandler/smallfull/TangibleAssetsHandlerTest.java

```diff
@@ -0,0 +1,171 @@
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.handler.Executor;
+import uk.gov.companieshouse.api.handler.smallfull.SmallFullResourceHandler;
+import uk.gov.companieshouse.api.handler.smallfull.tangible.TangibleResourceHandler;
+import uk.gov.companieshouse.api.handler.smallfull.tangible.request.TangibleCreate;
+import uk.gov.companieshouse.api.handler.smallfull.tangible.request.TangibleDelete;
+import uk.gov.companieshouse.api.handler.smallfull.tangible.request.TangibleGet;
+import uk.gov.companieshouse.api.handler.smallfull.tangible.request.TangibleUpdate;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.model.accounts.smallfull.SmallFullApi;
+import uk.gov.companieshouse.api.model.accounts.smallfull.SmallFullLinks;
+import uk.gov.companieshouse.api.model.accounts.smallfull.tangible.TangibleApi;
+import uk.gov.companieshouse.web.accounts.enumeration.NoteType;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.service.smallfull.SmallFullService;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class TangibleAssetsHandlerTest {
+
+    @Mock
+    private ApiClient apiClient;
+
+    @Mock
+    private SmallFullService smallFullService;
+
+    @Mock
+    private SmallFullResourceHandler smallFullResourceHandler;
+
+    @Mock
+    private TangibleResourceHandler tangibleAssetsResourceHandler;
+
+    @Mock
+    private TangibleGet tangibleAssetsGet;
+
+    @Mock
+    private TangibleUpdate tangibleAssetsUpdate;
+
+    @Mock
+    private TangibleCreate tangibleAssetsCreate;
+
+    @Mock
+    private TangibleDelete tangibleAssetsDelete;
+
+    @Mock
+    private TangibleApi tangibleAssetsApi;
+
+    @Mock
+    private SmallFullLinks smallFullLinks;
+
+    @Mock
+    private SmallFullApi smallFullApi;
+
+    @InjectMocks
+    private TangibleAssetsHandler tangibleAssetsHandler;
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+    private static final String TRANSACTION_ID = "transactionId";
+
+    private static final String URI = "/transactions/" + TRANSACTION_ID + "/company-accounts/" + COMPANY_ACCOUNTS_ID + "/small-full/note/tangible-assets";
+
+    private static final String TANGIBLE_ASSETS = "tangibleAssets";
+
+    @Test
+    @DisplayName("Get the resource URI")
+    void getResourceURI() {
+
+        assertEquals(URI, tangibleAssetsHandler.getUri(TRANSACTION_ID, COMPANY_ACCOUNTS_ID));
+    }
+
+    @Test
+    @DisplayName("Get CurrentAssetsInvestments Resource")
```

Comment:
Maybe best to replace instances of `CurrentAssetsInvestments` to `TangibleAssets`